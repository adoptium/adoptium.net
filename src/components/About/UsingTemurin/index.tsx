import React from "react"
import Image from "next/image"

const UsingTemurin = () => {
  return (
    <>
      <section className="py-16 max-w-[1048px] w-full mx-auto lg:flex justify-center lg:justify-between xl:px-0 px-6 ">
        <div className="lg:max-w-[560px] w-full">
          <h2 className="md:text-5xl text-[40px] font-normal leading-[48px] md:leading-[56px]">
            How other companies are using Temurin
          </h2>
            <p className="md:mt-6 my-10 text-grey text-base leading-6 font-normal">
            Discover how industry leaders leverage Temurin to enhance performance and drive innovation in their development processes.
            </p>
        </div>
        <div className="max-w-[400px] w-full hidden lg:block">
          <Image
            src="/images/icons/experience.png"
            className="mb-0"
            alt="scroll-divider"
            width={400}
            height={400}
          />
        </div>
      </section>
    </>
  )
}

export default UsingTemurin
