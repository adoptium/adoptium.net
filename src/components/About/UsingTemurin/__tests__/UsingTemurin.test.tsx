import React from "react"
import { render } from "@testing-library/react"
import { describe, expect, it } from "vitest"
import UsingTemurin from "../index"

describe("UsingTemurin component", () => {
  it("should render correctly", () => {
    const { container } = render(<UsingTemurin />)

    expect(container).toMatchSnapshot()
  })
})
