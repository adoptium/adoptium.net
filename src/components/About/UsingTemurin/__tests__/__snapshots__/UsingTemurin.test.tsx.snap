// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UsingTemurin component > should render correctly 1`] = `
<div>
  <section
    class="py-16 max-w-[1048px] w-full mx-auto lg:flex justify-center lg:justify-between xl:px-0 px-6 "
  >
    <div
      class="lg:max-w-[560px] w-full"
    >
      <h2
        class="md:text-5xl text-[40px] font-normal leading-[48px] md:leading-[56px]"
      >
        How other companies are using Temurin
      </h2>
      <p
        class="md:mt-6 my-10 text-grey text-base leading-6 font-normal"
      >
        Discover how industry leaders leverage Temurin to enhance performance and drive innovation in their development processes.
      </p>
    </div>
    <div
      class="max-w-[400px] w-full hidden lg:block"
    >
      <img
        alt="scroll-divider"
        class="mb-0"
        data-nimg="1"
        decoding="async"
        height="400"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Ficons%2Fexperience.png&w=828&q=75"
        srcset="/_next/image?url=%2Fimages%2Ficons%2Fexperience.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Ficons%2Fexperience.png&w=828&q=75 2x"
        style="color: transparent;"
        width="400"
      />
    </div>
  </section>
</div>
`;
